packages/terra-menu/src/MenuItem.scss

```diff
@@ -10,15 +11,29 @@
       &:focus {
         background-color: var(--terra-menu-item-hover-color, #ebf6fd);
       }
-    } 
+    }
+  }
+
+  .item:active:not(.is-disabled),
+  .active {
+    background-color: var(--terra-menu-item-active-color, #ebf6fd);
+
+    .title {
+      color: var(--terra-menu-item-active-text-color, #1c1f21);
```

Comment:
There's not a concrete standard of naming these variables but in general I believe they should be named only with the property and scoping. 

--terra-menu-item-active-**title**-color

Reply:
The prop name is actually `text` so I would keep this variable named as --terra-menu-item-active-text-color

Although probably out of the scope of this PR I think the css class declaration should be updated to be `.text` instead of `.title`